packages/contracts/contracts/voting/token/TokenVoting.sol

```diff
@@ -70,6 +78,10 @@ contract TokenVoting is MajorityVotingBase {
         uint256 totalVotingPower = votingToken.getPastTotalSupply(snapshotBlock);
         if (totalVotingPower == 0) revert NoVotingPower();
 
+        if (votingToken.getPastVotes(_msgSender(), snapshotBlock) < minProposerVotingPower) {
```

Comment:
get past votes? Shouldn't it be `votingPower` or similar?

Reply:
`getPastVotes` comes from the OZ's `IVotes`/`IVotesUpgradeable` governance token standard. We cannot change it